instat/dlgTransformText.vb

```diff
@@ -194,36 +198,35 @@ Public Class dlgTransformText
 
         clsPadFunction.SetPackageName("stringr")
         clsPadFunction.SetRCommand("str_pad")
-        clsPadFunction.AddParameter("pad", Chr(34) & " " & Chr(34))
-        clsPadFunction.AddParameter("side", Chr(34) & "left" & Chr(34))
-        clsPadFunction.AddParameter("width", 1)
+        clsPadFunction.AddParameter("width", 1, iPosition:=1)
+        clsPadFunction.AddParameter("side", Chr(34) & "left" & Chr(34), iPosition:=2)
+        clsPadFunction.AddParameter("pad", Chr(34) & " " & Chr(34), iPosition:=3)
 
         clsTrimFunction.SetPackageName("stringr")
         clsTrimFunction.SetRCommand("str_trim")
-        clsTrimFunction.AddParameter("side", Chr(34) & "left" & Chr(34))
+        clsTrimFunction.AddParameter("side", Chr(34) & "left" & Chr(34), iPosition:=1)
 
         clsWordsFunction.SetPackageName("stringr")
         clsWordsFunction.SetRCommand("word")
-        clsWordsFunction.AddParameter("start", 1)
-        clsWordsFunction.AddParameter("end", 2)
-        clsWordsFunction.AddParameter("sep", "stringr::fixed(" & Chr(34) & " " & Chr(34) & ")")
+        clsWordsFunction.AddParameter("start", 1, iPosition:=1)
+        clsWordsFunction.AddParameter("end", 2, iPosition:=2)
+        clsWordsFunction.AddParameter("sep", "stringr::fixed(" & Chr(34) & " " & Chr(34) & ")", iPosition:=3)
 
         clsSubstringFunction.SetPackageName("stringr")
         clsSubstringFunction.SetRCommand("str_sub")
-        clsSubstringFunction.AddParameter("start", 1)
-        clsSubstringFunction.AddParameter("end", 2)
+        clsSubstringFunction.AddParameter("start", 1, iPosition:=1)
+        clsSubstringFunction.AddParameter("end", 2, iPosition:=2)
 
         ucrBase.clsRsyntax.SetBaseRFunction(clsConvertFunction)
     End Sub
 
     Private Sub SetRCodeForControls(bReset As Boolean)
         bRCodeSet = False
+        ucrPnlPad.AddAdditionalCodeParameterPair(clsPadFunction, clsNewRParameter:=New RParameter("side", 2), iAdditionalPairNo:=1)

```

Comment:
We need to have the `AddAdditionalCodeParameterPair` before any of the `SetRCodes` run to avoid bugs